android/src/main/java/com/mrousavy/camera/CameraViewModule.kt

```diff
@@ -83,8 +82,7 @@ class CameraViewModule(reactContext: ReactApplicationContext) : ReactContextBase
 
   private fun findCameraView(viewId: Int): CameraView {
     Log.d(TAG, "Finding view $viewId...")
-    val mContext = WeakReference(reactApplicationContext)
-    val ctx = mContext?.get()
+    val ctx = reactApplicationContext
     val view = if (ctx != null) UIManagerHelper.getUIManager(ctx, viewId)?.resolveView(viewId) as CameraView? else null
```

Comment:
Can you remove `ctx` and just use `reactApplicationContext` directly? I see no point in declaring an extra variable.